package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,340 @@
+to perform Principal Component Analysis.
+
+.. _PCA-tutorial:
+The example uses files provided as part of the MDAnalysis test suite
+(in the variables :data:`~MDAnalysis.tests.datafiles.PSF` and
+:data:`~MDAnalysis.tests.datafiles.DCD`). This tutorial shows how to use the
+PCA class.
+
+First load all modules and test data ::
+    >>> import MDAnalysis as mda
+    >>> import MDAnalysis.analysis.pca as pca
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+
+Given a universe containing trajectory data we can perform PCA using
+:class:`PCA`:: and retrieve the principal components.
+    >>> u = mda.Universe(PSF,DCD)
+    >>> PSF_pca = pca.PCA(u, select='backbone')
+    >>> PSF_pca.run()
+
+Inspect the components to determine the principal components you would like
+to retain. The choice is arbitrary, but I will stop when 95 percent of the
+variance is explained by the components. This cumulated variance by the
+components is conveniently stored in the one-dimensional array attribute
+`cumulated_variance`. The value at the ith index of `cumulated_variance` is the
+sum of the variances from 0 to i.
+
+    >>> n_pcs = next(x[0] for x in enumerate(PSF_pca.cumulated_variance)
+    >>> if x[1] > 0.95)
+    >>> atomgroup = u.select_atoms('backbone')
+    >>> pca_space = PSF_pca.transform(atomgroup, n_components=n_pcs)
+
+From here, inspection of the pca_space and conclusions to be drawn from the
+data are left to the user.
+
+Functions
+---------
+.. autoclass:: PCA
+.. autofunction:: cosine_content
+"""
+from six.moves import range
+import logging
+import warnings
+
+import numpy as np
+
+from scipy.integrate import simps
+from MDAnalysis.core import AtomGroup
+from MDAnalysis import Universe
+from MDAnalysis.analysis.align import _fit_to
+from MDAnalysis.lib.log import ProgressMeter
+
+from .base import AnalysisBase
+
+
+class PCA(AnalysisBase):
+    """Principal component analysis on an MD trajectory
+
+    Attributes
+    ----------
+    p_components: array, (n_components, n_atoms * 3)
+        The principal components of the feature space,
+        representing the directions of maximum variance in the data.
+    variance : array (n_components, )
+        The raw variance explained by each eigenvector of the covariance
+        matrix.
+    cumulated_variance : array, (n_components, )
+        Percentage of variance explained by the selected components and the sum
+        of the components preceding it. If a subset of components is not chosen
+        then all components are stored and the cumulated variance will converge
+        to 1.
+    pca_space : array (n_frames, n_components)
+        After running :method:`pca.transform(atomgroup)` the projection of the
+        positions onto the principal components will exist here.
+    mean_atoms: MDAnalyis atomgroup
+        After running :method:`PCA.run()`, the mean position of all the atoms
+        used for the creation of the covariance matrix will exist here.
+    start: int
```

Comment:
you can remove them in the attribute section. They aren't interesting for the analysis, also they are variables from the base class and you don't explain the others like `n_frames`.